.gitignore

```diff
@@ -118,4 +118,7 @@ nCrunchTemp_*
 
 .fake
 *.pdb
-*.dll
\ No newline at end of file
+*.dll
+
+/test.fs
```

Comment:
Were those on purpose?

Reply:
Yes, those are produced during debugging. Because of [this](https://github.com/dotnet/fsharp/blob/539de8504bedaf51985ff5992b89b17d31b89618/vsintegration/tests/UnitTests/QuickInfoProviderTests.fs#L29), I guess.